### PR TITLE
scorep/30: ok on dom

### PR DESCRIPTION
**Pull request by jgphpc**
# DOM/2016.11
[x] HASWELL: [regression tests 783[0-3]](https://madra.cscs.ch/eth-cscs/regression/commit/aa11008197ffe583bad95c4dd2a9856bc6a9184e)
[x] BROADWELL: [regression tests 783[4-7]](https://madra.cscs.ch/eth-cscs/regression/commit/cb7a86c7e7f746c12faf8548728ed2cd6b6a1430)

### CODE REVIEW

**Comment by gppezzi** [comment]
@jgphpc let us know when this is ready to merge (from the subject line it is not clear)

**Comment by jgphpc** [comment]
intel + mpi/openmp + scorep/3 is broken (mpi only ok) => i opened a bug
but the team is busy at SC...
The other compilers are all ok.

**Comment by jgphpc** [comment]
ready to merge

File: easybuild/easyconfigs/c/Cube/Cube-4.3.4-CrayCCE-2016.11.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# !--- mandatory for dummy toolchain:
+#  module load Qt/5.7.0-debug
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://apps.fz-juelich.de/scalasca/releases/cube/%(version_major_minor)s/dist']
+checksums = [ '50f73060f55311cb12c5b3cb354d59fa' ]
+patches = [ 'Cube-4.3.4-qt57.patch', 'Cube-4.3.4-qt57isystem.patch' ]
+
+dependencies = [
+    ('Qt', '5.7.0-debug', '', True),
```
Comment (gppezzi) [inline easybuild/easyconfigs/c/Cube/Cube-4.3.4-CrayCCE-2016.11.eb]: where's the .eb file for `Qt/5.7.0-debug` ?

File: easybuild/easyconfigs/s/Score-P/Score-P-3.0-CrayPGI-2016.11.eb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+checksums = [ '44da8beaa3f71436a5f6fe51938aab2f' ]
+
+dependencies = [
+    ('libiberty', '5.3.0-scorep', '', True),
+    ('libunwind', '1.1-scorep', '', True),
+#    ('Cube', '4.3.4-debug', '', True),
+    ('Cube', '4.3.4'),
+    ('OPARI2', '2.0.1', '', True),
+#PR3501    ('OTF2', '2.0', '', True),
+#    ('SIONlib', '1.7.0l'),
```
Comment (gppezzi) [inline easybuild/easyconfigs/s/Score-P/Score-P-3.0-CrayPGI-2016.11.eb]: remove comment?

**Comment by gppezzi** [comment]
can you please provide a build report/log using this branch?

**Comment by gppezzi** [comment]
you can test the new eb setup script that uses a custom tools repository:
```
source /apps/common/UES/tools/easybuild/setup.sh-stable /path/to/sandbox /path/to/tools
```

File: easybuild/easyconfigs/c/Cube/Cube-4.3.4-CrayCCE-2016.11.eb (hunk 1 above)
Comment (jgphpc) [inline easybuild/easyconfigs/c/Cube/Cube-4.3.4-CrayCCE-2016.11.eb]: https://github.com/eth-cscs/tools/pull/42/commits/bbcebd2b622ca99d5cfedec44da67512faf17c72

File: easybuild/easyconfigs/c/CrayPGI/CrayPGI-2016.11.eb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+homepage = 'http://docs.cray.com/books/S-9408-1611/'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-pgi module (PE release: 11.2016).\n"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+dependencies = [
+    # PrgEnv version is not pinned, as Cray recommends to use the latest (default) version
+    ('PrgEnv-pgi', EXTERNAL_MODULE),
+    ('pgi/16.9.0', EXTERNAL_MODULE),
+    # ('cray-libsci/16.11.1', EXTERNAL_MODULE),
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/CrayPGI/CrayPGI-2016.11.eb]: 9ddce12b8c9bbfaf8575929c866a86ebe37a908d

**Comment by jgphpc** [comment]
# Logs
## broadwell
* cd /apps/common/UES/sandbox/jgp/easybuild.daint/broadwell/easybuild/software/
```
./CrayGNU/2016.11/easybuild/easybuild-CrayGNU-2016.11-20161121.173925.log
./CrayPGI/2016.11/easybuild/easybuild-CrayPGI-2016.11-20161121.174148.log
./CrayCCE/2016.11/easybuild/easybuild-CrayCCE-2016.11-20161121.174420.log
./CrayIntel/2016.11/easybuild/easybuild-CrayIntel-2016.11-20161121.174613.log

./Cube/4.3.4-CrayGNU-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.174143.log
./Cube/4.3.4-CrayPGI-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.174450.log
./Cube/4.3.4-CrayCCE-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.174702.log
./Cube/4.3.4-CrayIntel-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.174926.log

./Score-P/3.0-CrayGNU-2016.11/easybuild/easybuild-Score-P-3.0-20161121.174641.log
./Score-P/3.0-CrayCCE-2016.11/easybuild/easybuild-Score-P-3.0-20161121.175518.log
./Score-P/3.0-CrayPGI-2016.11/easybuild/easybuild-Score-P-3.0-20161121.175801.log
./Score-P/3.0-CrayIntel-2016.11/easybuild/easybuild-Score-P-3.0-20161121.180045.log
```


## haswell
* cd /apps/common/UES/sandbox/jgp/easybuild.daint/haswell/easybuild/software/
```
./CrayGNU/2016.11/easybuild/easybuild-CrayGNU-2016.11-20161121.182145.log
./CrayIntel/2016.11/easybuild/easybuild-CrayIntel-2016.11-20161121.182603.log
./CrayPGI/2016.11/easybuild/easybuild-CrayPGI-2016.11-20161121.182631.log
./CrayCCE/2016.11/easybuild/easybuild-CrayCCE-2016.11-20161121.182640.log

./Cube/4.3.4-CrayGNU-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.182424.log
./Cube/4.3.4-CrayIntel-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.182956.log
./Cube/4.3.4-CrayPGI-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.183011.log
./Cube/4.3.4-CrayCCE-2016.11/easybuild/easybuild-Cube-4.3.4-20161121.183013.log

./Score-P/3.0-CrayGNU-2016.11/easybuild/easybuild-Score-P-3.0-20161121.182814.log
./Score-P/3.0-CrayPGI-2016.11/easybuild/easybuild-Score-P-3.0-20161121.183719.log
./Score-P/3.0-CrayCCE-2016.11/easybuild/easybuild-Score-P-3.0-20161121.183608.log
./Score-P/3.0-CrayIntel-2016.11/easybuild/easybuild-Score-P-3.0-20161121.183737.log
```

**Comment by gppezzi** [comment]
qt does not compile on ELA or DOM with system gcc. 

Where does this recipe work and which gcc should we use?

**Comment by jgphpc** [comment]
Solved in https://github.com/eth-cscs/production/pull/5

**Pull request closed without merging**